packages/updatePrescriptionStatus/tests/utils/testUtils.ts

```diff
@@ -48,6 +49,7 @@ const TASK_VALUES = [
     id: TASK_ID_1,
     status: "in-progress",
     businessStatus: "Ready to collect",
+    repeatNo: 1,
```

Comment:
Removing this line makes the tests fail.

Reply:
Tests fail when `repeatNo` field removed from both objects in `TASK_VALUES`

Reply:
The optional field `repeatNo` has been removed.